substrate/client/transaction-pool/src/fork_aware_txpool/tx_mem_pool.rs

```diff
@@ -344,8 +344,12 @@ where
 				Err(sc_transaction_pool_api::error::Error::AlreadyImported(Box::new(hash))),
 			(true, _) => Err(sc_transaction_pool_api::error::Error::ImmediatelyDropped),
 		};
-		log::trace!(target: LOG_TARGET, "[{:?}] mempool::try_insert: {:?}", hash, result.as_ref().map(|r| r.hash));
-
+		tracing::trace!(
+			target: LOG_TARGET,
+			?hash,
```

Comment:
```suggestion
			tx_hash = ?hash,
```
... or rename `hash` to `tx_hash` (we need same `tx_hash` field in all logs which refer to single transcation.